### ISSUE TITLE
cs-CS language is being changed by cs-RS

### ISSUE THREAD

**Issue by Tominka** [Romanchuk/angular-i18next#129]
Hello,

since i have been using Angular 18, i´ve got issue with languge key cs-CS.
Library is finding loaded cs-CS sources under cs-RS language.

Testing code:
![image](https://github.com/user-attachments/assets/1200a3e8-fde5-4b56-8f49-e841f4ac35f8)

Console output:
![image](https://github.com/user-attachments/assets/9cf84249-3985-4f17-9296-624e489223c1)

I tried a lot of ways. Only way is change cs-CS across the project to cs-RS, but **it is not solution** an i need to have it correctly.
Should it be a library bug?

**Comment by adrai** [comment]
https://github.com/i18next/i18next/issues/2258

**Comment by Romanchuk** [comment]
I don't think this is related to this package. This package is a wrapper around i18next and ideology is "not override an original library behaviour"

**Comment by Tominka** [comment]
> [i18next/i18next#2258](https://github.com/i18next/i18next/issues/2258)

It solved that problem. I didn't think of that. 

Thank you.

I´m closing this issue.
